docs: update readme (#15)
logos-co/logos-delivery-module: README.md

nix build '.#default'
```

The result will include:

> - `/lib/delivery_module_plugin.dylib` (or `.so` on Linux) - The Delivery module plugin (liblogosdelivery is linked statically)

Reviewer: No, its not. liblogosdelivery is dynamically linked.
```suggestion
- `/lib/delivery_module_plugin.dylib` (or `.so` on Linux) - The Delivery module plugin
- `/lib/liblogosdelivery.dylib` (or `.so` on linux) - The logos-delivery library
- `/lib/librln.dylib` (or `.so` in linux) - Zerokit's RLN library
```

```
result/
└── lib/
    ├── delivery_module_plugin.dylib  # or .so on Linux — Logos module plugin

>     │                                 #   (liblogosdelivery is linked statically)

Reviewer: ```suggestion
    ├── liblogosdelivery.dylib
    ├── librln.dylib
```

- Qt6 Remote Objects (qtremoteobjects)
- logos-liblogos (provided via Nix)
- logos-cpp-sdk (provided via Nix)

> - logos-delivery / liblogosdelivery — linked statically (provided via Nix)

Reviewer: ```suggestion
- logos-delivery / liblogosdelivery — target (provided via Nix)
```
